### PR TITLE
feature/overriding-websocket-connection: I needed to conect to a serv…

### PR DESCRIPTION
**Pull request by antoniolazaro**
…er without inform a port and I couldn't, so I made this version to this scenario

### CODE REVIEW

File: src/main/java/com/surrealdb/connection/SurrealWebSocketConnection.java
Comment (phughk) [inline src/main/java/com/surrealdb/connection/SurrealWebSocketConnection.java:55]: The host string should support URI format. Happy with default ports, but the implementation here is doing protocol prefixing - I would like to avoid that please

**Review by phughk — COMMENTED** [review]
Thanks for opening this! Added a comment, but would be happy to approve if you parse the URI and set a default port if one wasnt provided. This may be possible to find I believe in a parsed URI

**Review by phughk — APPROVED** [review]
Thanks!

**Comment by allangomessl** [comment]
Hey guys, there is any update here?
I need this feature. 🫠

**Comment by oftoapp** [comment]
@phughk, @antoniolazaro Please, could you merge this PR?

**Comment by antoniolazaro** [comment]
> @phughk, @antoniolazaro Please, could you merge this PR?

I can't merge.

1 workflow awaiting approval
This workflow requires approval from a maintainer. [Learn more about approving workflows.](https://docs.github.com/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks)

**Comment by allangomessl** [comment]
The last commit was 10 months ago, this library is completely abandoned 🫠

**Comment by phughk** [comment]
Heya, please open PRs against https://github.com/surrealdb/surrealdb.java/tree/emmanuel/jni
@emmanuel-keller is doing a rewrite
Closing this PR as it's not necessary due to the change of direction

**Comment by allangomessl** [comment]
thank you @phughk!

There is a roadmap or ETA to be released?

For know, I want to use surrealdb to store my application logs, I can test if a alpha version was published on maven central.

**Pull request closed without merging**
